types/mongodb/tslint.json

```diff
@@ -12,6 +12,8 @@
         "no-consecutive-blank-lines": false,
         "no-padding": false,
         "no-unnecessary-generics": false,
+        "no-redundant-jsdoc": false,
```

Comment:
If this was to work around a build failure due to `@deprecated`, I think that's been fixed in dtslint and these shouldn't be necessary anymore.

Reply:
It's still happening :(

How can I fix that?
```
$ npm run lint mongodb 

> definitely-typed@0.0.3 lint /Users/TJT/Projects/DefinitelyTyped
> dtslint types "mongodb"

The 'no-redundant-jsdoc-2' rule threw an error in '/Users/TJT/Projects/DefinitelyTyped/types/mongodb/index.d.ts':
Error: Unexpected tag kind: JSDocDeprecatedTag
    at checkTag (/Users/TJT/Projects/DefinitelyTyped/node_modules/dtslint/bin/rules/noRedundantJsdoc2Rule.js:118:23)
    at cb (/Users/TJT/Projects/DefinitelyTyped/node_modules/dtslint/bin/rules/noRedundantJsdoc2Rule.js:67:25)
    at visitNodes (/Users/TJT/Projects/DefinitelyTyped/node_modules/typescript/lib/typescript.js:27024:30)
    at Object.forEachChild (/Users/TJT/Projects/DefinitelyTyped/node_modules/typescript/lib/typescript.js:27335:21)
    at cb (/Users/TJT/Projects/DefinitelyTyped/node_modules/dtslint/bin/rules/noRedundantJsdoc2Rule.js:72:19)
    at Array.forEach (<anonymous>)
    at walk (/Users/TJT/Projects/DefinitelyTyped/node_modules/dtslint/bin/rules/noRedundantJsdoc2Rule.js:56:34)
    at Rule.AbstractRule.applyWithFunction (/Users/TJT/Projects/DefinitelyTyped/node_modules/tslint/lib/language/rule/abstractRule.js:39:9)
    at Rule.apply (/Users/TJT/Projects/DefinitelyTyped/node_modules/dtslint/bin/rules/noRedundantJsdoc2Rule.js:36:21)
    at Linter.applyRule (/Users/TJT/Projects/DefinitelyTyped/node_modules/tslint/lib/linter.js:214:29)
```

Reply:
Can you check what version of dtslint you have locally via `npm ls dtslint`? The latest version is 3.6.14.

Reply:
Oh thanks, the problem fixed!